alchemy/src/destroy.ts

```diff
@@ -127,10 +133,10 @@ export async function destroy<Type extends string>(
           parent: scope,
         },
         async (scope) => {
-          nestedScope = scope;
+          nestedScope = options?.replace?.props == null ? scope : undefined;
```

Comment:
Why are we checking null? Does it mean anything different than undefined?

Reply:
in this case, there is no difference
force of habit to do this whenever I want to check null or undef over
`if(value)` which can check falsey values
`if(value === undefined)` or `if(value === null)` which only checks one and can slip pass ts if you're not careful since TS won't error on
```
const test: string | undefined = undefined;
if(test === null) {}
```
even though test will never be null and that will always fail

Reply:
LMK if you dislike it and I can stop